src/Agents/Dhadgar.Agent.Windows/Installer/README.md

```
@@ -0,0 +1,228 @@
+```
+
+## Properties
+
+| Property | Description | Default |
+|----------|-------------|---------|
+| `INSTALLFOLDER` | Installation directory | `C:\Program Files\Meridian Console\Agent\` |
+| `ENROLLMENT_TOKEN` | One-time enrollment token from control plane | (empty) |
+
+## What Gets Installed
+
+### Files
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)